From c796485cc15598a78e6dbc88dc086c5dd15e332f Mon Sep 17 00:00:00 2001
From: Simon Busch <morphis@gravedo.de>
Date: Thu, 20 Sep 2012 09:35:53 +0200
Subject: [PATCH] Add a common linux target which uses .so as suffix instead
 of .module

Open-webOS-DCO-1.0-Signed-off-by: Simon Busch <morphis@gravedo.de>
---
 src/target/common/CMakeLists.txt |   28 ++++++++++++++++++++++++++++
 1 file changed, 28 insertions(+)
 create mode 100644 src/target/common/CMakeLists.txt

diff --git a/src/target/common/CMakeLists.txt b/src/target/common/CMakeLists.txt
new file mode 100644
index 0000000..99c0147
--- /dev/null
+++ b/src/target/common/CMakeLists.txt
@@ -0,0 +1,28 @@
+# @@@LICENSE
+#
+#      Copyright (c) 2010-2012 Hewlett-Packard Development Company, L.P.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# LICENSE@@@
+
+set (CMAKE_SHARED_MODULE_PREFIX "nyx")
+set (CMAKE_SHARED_MODULE_SUFFIX ".so")
+# Use multiarch paths
+if(CMAKE_SIZEOF_VOID_P EQUAL 8)
+set (NYX_MODULE_DIR "/lib/aarch64-linux-gnu/nyx/modules")
+set (NYX_MODULE_MOCK_DIR "/lib/aarch64-linux-gnu/nyx/modules.mock")
+else()
+set (NYX_MODULE_DIR "/lib/arm-linux-gnueabi/nyx/modules")
+set (NYX_MODULE_MOCK_DIR "/lib/arm-linux-gnueabi/nyx/modules.mock")
+endif()
-- 
1.7.9.5

